test/yaml-rest-runner/src/main/java/org/elasticsearch/test/rest/yaml/section/DoSection.java

```diff
@@ -363,7 +363,7 @@ public void execute(ClientYamlTestExecutionContext executionContext) throws IOEx
             final String testPath = executionContext.getClientYamlTestCandidate() != null
                 ? executionContext.getClientYamlTestCandidate().getTestPath()
                 : null;
-            if (executionContext.esVersion().major == Version.V_7_17_0.major && executionContext.esVersion().after(Version.V_7_17_1)) {
+            if (executionContext.esVersion().after(Version.V_7_17_1)) {
```

Comment:
This is fixed in a separate PR, will remove it in the final revision; it's here to check the fix works :)